build.gradle.kts

```
@@ -0,0 +1,84 @@
+import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
+
+plugins {
+    id("com.android.application") version "8.0.0" apply false
+    id("com.android.library") version "8.0.0" apply false
+    id("org.jetbrains.kotlin.android") version "1.8.20" apply false
+    id("com.google.gms.google-services") version "4.3.15" apply false
+    id("com.google.firebase.crashlytics") version "2.9.4" apply false
+    id("com.google.firebase.firebase-perf") version "1.4.2" apply false
+    id("androidx.navigation.safeargs") version "2.5.3" apply false
+    id("com.github.ben-manes.versions") version "0.41.0" apply false
```

Turns out we need to apply this at the module level.

```suggestion
    id("com.github.ben-manes.versions") version "0.41.0" apply true
```